web/modules/custom/helloworld/helloworld.module

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * Contains the hook impletemnations.
+ */
+
+ /**
+  * Implements hook_permission().
+  */
+function helloworld_permission() {
+  return array(
+    'administer hello module' => array(
+      'title' => t('Administer my module'),
+      'description' => t('Perform administration tasks for my module.'),
+    ),
+  );
```

Comment:
@Vishalprasad28 Did you run your code through PHPCS before raising the PR?

Reply:
Yes sir and it doesn't show any error other than for t() function, but how can we use $this->t() inside hook hor any other class than doesn't extend or implement and drupal core class, even drupal vore uses the t() functions at many places